### DISCUSSION TITLE
Doing a semester abroad, can't access GitHub Education benefits

### DISCUSSION THREAD

**Discussion by Luchassmed** [community/community#138251] [GitHub Education]
### Hi everyone,

I am trying to request education benefits, which I typically do at the start of every fall semester without any issues. However, this fall semester, I am studying abroad, and GitHub was unable to approve my educational discount request because my school I selected, obviously, doesn't appear to have a campus location in the country where I am currently abroad.

Has anyone experienced something similar and found a workaround?

**Comment by alx-mp4** [comment]
Hi @Luchassmed, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
